src/style.css

```diff
@@ -0,0 +1,183 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+}
+
+.cont-navbar {
+  display: flex;
+  align-items: center;
+  width: 75%;
+  margin: 0 auto;
+  padding: 1rem 0.5rem;
+}
+
+.cont-link {
+  margin-left: 2rem;
+}
+
+.cont-link-lists {
+  display: flex;
+  list-style: none;
+}
+
+.cont-link-lists > *:not(:last-child) {
+  margin-right: 2rem;
+}
+
+.link {
+  text-decoration: none;
+  font-size: 1.5rem;
+  padding: 0.5rem;
+}
+
+.link:active,
+.link:hover {
+  border-bottom: 2px solid currentColor;
+}
+
+.main {
+  width: 80%;
+  margin: 0 auto;
+  display: grid;
+  grid-template-rows: repeat(3, auto);
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 3rem;
+}
+
+.cont-img {
+  width: 100%;
+  text-align: center;
+}
+
+.img-product {
+  height: 10rem;
+  margin: 0 auto;
+}
+
+.cont-info-wrapper {
+  width: 100%;
+  display: flex;
+}
```

Comment:
- [ ] [OPTIONAL] if you would like to make your project look even better it might look nice if you change the placement of the like button, right now it is aligned with the title of the product which makes it shift on the page depending on the size of the title. if all like buttons are aligned in the same place it will make your project look a lot more uniform.